bazel/external/wee8.genrule_cmd

```diff
@@ -70,10 +70,19 @@ WEE8_BUILD_ARGS+=" v8_use_external_startup_data=false"
 # Disable read-only heap, since it's leaky and HEAPCHECK complains about it.
 # TODO(PiotrSikora): remove when fixed upstream.
 WEE8_BUILD_ARGS+=" v8_enable_shared_ro_heap=false"
+# Support Arm64
+if [[ `uname -m` == "aarch64" ]]; then
+  WEE8_BUILD_ARGS+=" target_cpu=\"arm64\""
+fi
 
 # Build wee8.
-third_party/depot_tools/gn gen out/wee8 --args="$$WEE8_BUILD_ARGS"
-third_party/depot_tools/ninja -C out/wee8 wee8
+if [[ `uname -m` == "aarch64" ]]; then
+  gn gen out/wee8 --args="$$WEE8_BUILD_ARGS"
+  ninja -C out/wee8 wee8
+else
+  third_party/depot_tools/gn gen out/wee8 --args="$$WEE8_BUILD_ARGS"
+  third_party/depot_tools/ninja -C out/wee8 wee8
```

Comment:
We don't really need to depend on the bundled tools, and we already have a dependency on `ninja`, so perhaps instead adding workarounds here, we should add a dependency on `gn` and use the tools from the `$PATH`?

What do you think, @lizan?

Reply:
Hi, @PiotrSikora 
Do you mean third party tools could be replaced with the system tool no matter x86 or arm64 ?

Reply:
Correct.

Reply:
@lizan  What is your opinion ?

Reply:
But we don't have gn installed today I think? Either way is fine to me.

Reply:
We don't, but we can add it as a requirement. Alternatively, we could test for its existence at runtime, and fallback to `depot_tools` in case it isn't available on the host.

Reply:
@lizan @PiotrSikora Could we add those tools into envoy-build-ubuntu image ?